contracts/axelar-gateway/src/contract.rs

```diff
@@ -223,6 +223,16 @@ impl AxelarGatewayInterface for AxelarGateway {
     fn epoch(env: &Env) -> u64 {
         auth::epoch(env)
     }
+
+    fn version(env: Env) -> Vec<u32> {
+        vec![&env, 0, 0, 1]
```

Comment:
we should get these from cargo env
https://github.com/axelarnetwork/axelar-amplifier/blob/main/contracts/axelarnet-gateway/src/contract.rs#L18
you can get the major minor individually as well i believe

Reply:
I was able to get:
```
axelar_gateway::contract 
const CONTRACT_VERSION: &str = "0.1.0"
```
updated.